src/patterns/ethnic-group/index.md.njk

```diff
@@ -109,14 +111,20 @@ Under the General Data Protection Regulation, there are different [rules around
 
 ## Research on this pattern
 
-This pattern was originally contributed by the Race Disparity Unit based on work and research findings from the Office of National Statistics (ONS).
+This pattern was originally contributed by the Race Disparity Unit.
+
+The examples and guidance here are based on [examples of ways to ask about ethnic groups](https://designnotes.blog.gov.uk/2019/01/29/researching-how-we-ask-users-about-their-ethnicity/) from a number of government services, and [research from the Office of National Statistics (ONS)](https://history.blog.gov.uk/2019/03/07/50-years-of-collecting-ethnicity-data/).
+
+The Race Disparity Unit researched this pattern as part of a prototype. They carried out pop-up research with over 30 participants with different ethnic backgrounds.
+
+Almost all participants answered the question quickly and easily, with no one backtracking to change their selection from the previous page.
 
-The question ‘What is your ethnic group?’ was agreed following a 2 year cross-government consultation led by the ONS.
+5 of the participants used the text input provided to specify 'another' ethnicity than those provided, which validates the need for this option.
 
-ONS research found that presenting all of the ethnic group options on a single page led to some users trying to select more than 1 option. Splitting the question across 2 pages solved this problem across multiple testing rounds with more than 300 users.
+2 participants chose not to answer the question after a step was introduced asking whether users were willing to provide their ethnic group.
 
-### Next steps
+## Next steps
 
-More research is needed to understand where to ask this question in relation to a service.
+More research is needed to understand where to ask this question in a service.
 
-If you've used this pattern in your service, please get in touch to share your research findings.
+If you’ve used this pattern in your service, please get in touch to share your research findings
```

Comment:
Was it intentional to drop the full stop at the end of this line?

Reply:
@joelanman Think this was just a mistake. Have fixed. 👍